orcid-web/src/main/java/org/orcid/frontend/web/controllers/RegistrationController.java

```diff
@@ -236,19 +236,19 @@ public void validateGrcaptcha(HttpServletRequest request, @RequestBody Registrat
 
         boolean usedCaptcha = false;
 
-        // If recatcha wasn't loaded do nothing. This is for countries that
-        // block google.
-        if (reg.getGrecaptchaWidgetId().getValue() != null) {
-            // If the captcha verified key is not in the session, redirect to
-            // the login page
-            if (request.getSession().getAttribute(GRECAPTCHA_SESSION_ATTRIBUTE_NAME) == null || PojoUtil.isEmpty(reg.getGrecaptcha())
-                    || !reg.getGrecaptcha().getValue().equals(request.getSession().getAttribute(GRECAPTCHA_SESSION_ATTRIBUTE_NAME))) {
-                r.setUrl(getBaseUri() + "/register");
-                return r;
-            }
-
-            usedCaptcha = true;
-        }
+        // If recatcha wasn't loaded do nothi
```

Comment:
@DanielPalafox why did you commented this code?

Reply:
@amontenegro Oh my mistake. Done